src/content/en/2025/accessibility.md

```diff
@@ -3,18 +3,1190 @@
+
+The Americans with Disabilities Act (ADA), which <a hreflang="en" href="https://www.access-board.gov/news/2025/07/21/celebrating-35-years-of-americans-with-disabilities-act/">turned 35 in July</a>, was pioneering work that set a precedent globally.
+
+<a hreflang="en" href="https://archive.opengovasia.com/2025/03/11/digital-accessibility-singapores-commitment-to-inclusivity/?c=ca">Singapore's recent commitment to accessibility</a> improvement, demonstrated through the open source tool <a hreflang="en" href="https://github.com/GovTechSG/oobee">Oobee</a>, shows emerging global momentum. Oobee allows organizations to scan hundreds of pages and generate consolidated accessibility reports, positioning it as a <a hreflang="en" href="https://www.digitalpublicgoods.net/r/oobee">Digital Public Good</a>.
+
+{{ figure_markup(
+  image="most-accessible-us-state-governments.png",
+  caption="The most accessible US state governments.",
+  description="A bar chart ranking US state and territory government sites by average accessibility score. New Hampshire leads at 94.61%, followed by Nevada (92.43%), Kansas (91.67%), New York (91.45%), and South Carolina (91.07%). Many other states, including Arizona, California, Missouri, Montana, and North Carolina, also score around 90% or higher, while states like New Jersey (83.91%) and Rhode Island (83.31%) appear toward the lower end of the list.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQFD-7C6Jv6q1JyviDsKosRlVwaok7g7nRCQ9NGMw5MaAAohL7EcDejVwgp13Z_T2S_57Zi0YaVb7st/pubchart?oid=445029634&format=interactive",
+  sheets_gid="1855027591",
+  sql_file="lighthouse_score_by_government.sql",
+  width=600,
+  height=500
+  )
+}}
+
+### Content Management Systems (CMS)
+
+A website's choice of Content Management System (CMS) significantly influences its accessibility outcomes. Using <a hreflang="en" href="https://almanac.httparchive.org/en/2024/methodology#wappalyzer">Wappalyzer</a> data, the 2025 Web Almanac compared accessibility scores across traditional CMSs, platforms, and specialized website builders. This revealed both consistent patterns and notable outliers.
+
+Among traditional CMSs, <a hreflang="en" href="https://www.sitecore.com/">Sitecore</a> maintained 85% accessibility in 2025, though its instance count dropped below 10,000. <a hreflang="en" href="https://business.adobe.com/products/experience-manager/sites/aem-sites.html">Adobe Experience Manager (AEM)</a> and <a hreflang="en" href="https://www.contentful.com/">Contentful</a> continue to lead, likely because larger corporations adopting these enterprise solutions have more resources to address accessibility issues. <a hreflang="en" href="https://wordpress.com/">WordPress</a> showed no significant improvement from 2024, but rose to third place, reflecting its market dominance and the growing accessibility consciousness of its user base.
+
+{{ figure_markup(
+  image="most-accessible-traditional-cms.png",
```

Comment:
I updated the percentage to remove decimal places as think it makes it more difficult to read.

Also is the X-Axis label ("Percent of input elements") correct? I think it's average Lighthouse score or something like that?

Reply:
Urgh. Guess I should fix that too then!

Yeah I just checked the SQL and it’s definitely average accessibility score.